cmd/ci-secret-bootstrap/main.go

```diff
@@ -0,0 +1,240 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/sirupsen/logrus"
+
+	coreapi "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	coreclientset "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+
+	"github.com/openshift/ci-tools/pkg/bitwarden"
+	"github.com/openshift/ci-tools/pkg/util"
+)
+
+type options struct {
+	dryRun         bool
+	kubeConfigPath string
+	configPath     string
+	bwUser         string
+	bwPasswordPath string
+
+	bwPassword    string
+	secretClients map[string]secretClient
+	config        config
+}
+
+type secretClient struct {
+	secretsGetter coreclientset.SecretsGetter
+	config        rest.Config
+}
+
+func parseOptions() options {
+	var o options
+	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+	fs.BoolVar(&o.dryRun, "dry-run", true, "Whether to actually create the secrets with oc command")
+	fs.StringVar(&o.kubeConfigPath, "kubeconfig", "", "Path to the kubeconfig file to use for CLI requests.")
+	fs.StringVar(&o.configPath, "config", "", "Path to the config file to use for this tool.")
+	fs.StringVar(&o.bwUser, "bw-user", "", "Username to access BitWarden.")
+	fs.StringVar(&o.bwPasswordPath, "bw-password-path", "", "Path to a password file to access BitWarden.")
+	if err := fs.Parse(os.Args[1:]); err != nil {
+		logrus.WithError(err).Errorf("cannot parse args: '%s'", os.Args[1:])
+	}
+	return o
+}
+
+func (o *options) validateOptions() error {
+	if o.bwUser == "" {
+		return fmt.Errorf("the mandatory arg bw-user is empty")
```

Comment:
`s/mandatory arg bw-user/--bw-user/` -- these are flags, not args, anyway

Reply:
Done.